feg/gateway/services/session_proxy/credit_control/gx/model_conversion_test.go

```diff
@@ -33,9 +33,12 @@ func TestReAuthRequest_ToProto(t *testing.T) {
 	// Check nil, 1-element, multiple elements, and empty arrays
 	monitoringKey := []byte("monitor")
 	monitoringKey2 := []byte("monitor2")
+	monitoringKey3 := []byte("monitor3")
 	bearerID := "bearer1"
 	var ratingGroup uint32 = 42
 	var totalOctets uint64 = 2048
+	var monitorSupport_0 gx.MonitoringSupport = 0x0
```

Comment:
Can we use the const values defined above here? Or is there a reason why we're not using it?

Reply:
😅
I was playing with the values and forgot to change it back
Thanks!